cmd/end2endtest/helpers.go

```diff
@@ -438,71 +455,112 @@ func (t *e2eElection) overwriteVote(choices []int, indexAcct int, waitType strin
 }
 
 // ballotVotes from a default list of 10 vote values that exceed the max value, max total cost and is not unique will
-func ballotVotes(b ballotData, nvotes int) ([][]int, [][]*types.BigInt) {
+func ballotVotes(vop vapi.TallyMode, nvotes int, ballotType string, expectUniqueV bool) ([][]int, [][]*types.BigInt) {
 	votes := make([][]int, 0, nvotes)
-	var resultsField1, resultsField2 []*types.BigInt
-
-	// initial 10 votes to be sent by the ballot test
-	var v = [][]int{
-		{0, 0}, {0, 7}, {5, 7}, {2, 0}, {2, 3},
-		{1, 6}, {0, 8}, {6, 5}, {2, 7}, {6, 4},
+	var resultsFields [][]*types.BigInt
+	var v [][]int
+
+	switch ballotType {
+	// fill initial 10 votes to be sent by the ballot test
+	case "ranked":
+		v = [][]int{
+			{0, 5, 0, 2}, {3, 1, 0, 2}, {3, 2, 0, 1}, {1, 0, 3, 2},
+			{2, 3, 1, 1}, {4, 1, 1, 0}, {0, 0, 2, 1}, {0, 2, 1, 3},
+			{1, 1, 1, 4}, {0, 3, 2, 1},
+		}
+		// default results on field 1,2,3,4 for 10 votes
+		resultsFields = append(resultsFields, votesToBigInt(20, 10, 0, 20),
+			votesToBigInt(10, 10, 20, 10), votesToBigInt(20, 10, 10, 10),
+			votesToBigInt(0, 20, 20, 10))
```

Comment:
i'd also refactor this, so that it's part of the Run (or Setup?) of the relevant test

Reply:
also, i'd move the whole ballotVotes func to ballot.go, and only leave in helpers.go funcs that are used by more than one test group (such as endElectionAndFetchResults)

but right now, helpers.go is full of funcs that are only used by one specific test. so i'd move each of those funcs to its corresponding test as well (including ballotVotes), in a PR specific to do that